build.gradle

```diff
@@ -8,62 +8,37 @@ plugins {
 	id 'idea'
 }
 
-sourceSets {
-	main {
-		java {
-			srcDir 'src/'
-		}
-	}
-	test {
-		java {
-			srcDir 'test-src/'
-		}
-		resources {
-			srcDir 'test-data/'
-		}
-	}
-
+ext {
+	teamscaleVersion = '5.9.0-rc3'
 }
 
- jar {
-	exclude ("**/CheckTest*")
- }
-
 test {
 	useJUnitPlatform()
+
 	testLogging {
 		// It's sometimes useful to see the stdout during testing
 		showStandardStreams = true
 		exceptionFormat = 'full'
 	}
 }
 
-
 repositories {
 	mavenCentral()
+
+	maven {
+		url "https://share.cqse.eu/svn/artifacts/maven"
+	}
 }
 
 dependencies {
-	implementation fileTree(dir: 'lib', exclude: '*-test-utils.jar')
+	implementation platform("com.teamscale:teamscale-bom:$teamscaleVersion")
+
+	implementation 'com.teamscale:teamscale-check-api'
+	implementation 'com.teamscale:teamscale-commons'
+	implementation 'com.teamscale:teamscale-lib-commons'
 	compile 'org.apache.logging.log4j:log4j-api:2.12.1'
-	runtimeOnly 'org.apache.commons:commons-compress:1.19'
-	runtimeOnly 'com.google.guava:guava:28.1-jre'
-	runtimeOnly 'com.thoughtworks.xstream:xstream:1.4.11.1'
-	runtimeOnly 'com.fasterxml.jackson.core:jackson-databind:2.10.1'
-	runtimeOnly 'org.apache.httpcomponents:httpmime:4.5.10'
-	runtimeOnly 'org.eclipse.jgit:org.eclipse.jgit:5.5.1.201910021850-r'
-	runtimeOnly 'org.eclipse.jgit:org.eclipse.jgit.ssh.apache:5.5.1.201910021850-r'
-	runtimeOnly 'com.jcraft:jsch:0.1.55'
-	runtimeOnly 'org.apache.commons:commons-lang3:3.9'
-	runtimeOnly 'org.apache.logging.log4j:log4j-core:2.12.1'
-	runtimeOnly 'jaxen:jaxen:1.2.0'
 
-	testImplementation fileTree(dir: 'lib', include: '*-test-utils.jar')
-	testImplementation 'org.junit.jupiter:junit-jupiter:5.5.2'
-	testImplementation 'org.junit.jupiter:junit-jupiter-engine:5.5.2'
-	testImplementation 'org.assertj:assertj-core:3.13.2'
-	testImplementation 'nl.jqno.equalsverifier:equalsverifier:3.1.10'
-	testImplementation 'org.junit.platform:junit-platform-launcher:1.5.2'
+	testImplementation 'com.teamscale:teamscale-check-api-test-fixtures'
 }
 
 eclipse {

```

Comment:
Large parts of this eclipse block are no longer needed AFAICS. I guess the first two ifs can be removed and the other one needs to be adjusted to the new layout